src/components/Home.js

```diff
@@ -32,7 +32,7 @@ function Home() {
             {filter.map((item) => (
               <NavLink className="list-info" to={`/${item.id}`} key={item.id}>
                 <label key={item.id} htmlFor={item.id}>
-                  <img src={item.icon} alt={item.id} />
+                  <img className="home-img" src={item.icon} alt={item.id} />
```

Comment:
- [ ] **Required:** Some of your images are out of proportion. Kindly fix them and re-deploy your app so the changes can be reflected.

![image](https://user-images.githubusercontent.com/27709832/217904479-97c39abe-6954-4a96-8669-f542ac7b67b7.png)

![image](https://user-images.githubusercontent.com/27709832/217904525-6820ca6e-2917-4bca-9d40-d925e673a3e5.png)


![image](https://user-images.githubusercontent.com/27709832/217904562-0f112ba5-587a-417d-8c9e-bc00311cf263.png)

Reply:
Sorry @brytebee I made a mistake, and the deploy live was not showing the latest version of the project... to do that I need to merge this PR and create a new one from dev to main... can I do that?... Please let me know, thanks!

Reply:
It is usually unacceptable to merge a PR that is still being reviewed. However, if you're sure that merging it would fix the current issues, go ahead. Just remember to link to this PR in the new one you'll create so that the next reviewer can follow the thread.